Library/Formula/autoconf-archive.rb

```diff
@@ -1,13 +1,20 @@
-require "formula"
-
 class AutoconfArchive < Formula
-  homepage "http://savannah.gnu.org/projects/autoconf-archive/"
+  homepage "https://savannah.gnu.org/projects/autoconf-archive/"
   url "http://ftpmirror.gnu.org/autoconf-archive/autoconf-archive-2014.10.15.tar.xz"
-  mirror "http://ftp.gnu.org/gnu/autoconf-archive/autoconf-archive-2014.10.15.tar.xz"
+  mirror "https://ftp.gnu.org/gnu/autoconf-archive/autoconf-archive-2014.10.15.tar.xz"
   sha1 "7efcefd29a67da2a7243ea2b30e353027d70b460"
 
   def install
+    # gnome_common shares two files with autoconf-archive.
+    # Both formulae too common/necessary to conflict or keg.
+    gnome_common = Pathname.new("#{Formula["gnome-common"].opt_prefix}")
```

Comment:
opt_prefix is already a pathname, the conditional can just be `if Formula["gnome-common"].opt_prefix.exist?`